ops/docker/ci-builder/Dockerfile

```diff
@@ -114,16 +117,9 @@ RUN echo "downloading and verifying Codecov uploader" && \
   rm codecov
 
 # within docker use bash
-SHELL [ "/bin/bash", "-c" ]
+SHELL ["/bin/bash"]
```

Comment:
Are we sure it's without `-c`? I think bash will then interpret the argument as a filename, to run a script, rather than interpret the arguments.